### ISSUE TITLE
Flat label list tab

### ISSUE THREAD

**Issue by agnanas** [biigle/largo#170]
When editing image or video, we have the "Annotations" tab in the sidebar that list all annotations marked in the current image/video. This is extremely handy. Would be great if this tab was also available in the Largo tool. When dealing with recursively label trees because it is very hard to spot what labels are present unless we open a second window to know what to labels to search for.

**Comment by mzur** [comment]
Thank you for the suggestion! What do you think about showing labels without annotations in a muted text color like this?

![image](https://github.com/biigle/largo/assets/2457311/19f48cd4-707a-4298-98d1-a56fbde2e209)

**Comment by dlangenk** [comment]
Can you dim it a little more? The difference is not that big in your example image. But this seems like a nice improvement.

**Comment by mzur** [comment]
> Can you dim it a little more?

No :smile: This is the "muted text" color that is used everywhere else in BIIGLE, too.

@agnanas What is your opinion?

**Comment by agnanas** [comment]
That does make it prominent in a stand alone label tree structure. But in a recursive label tree (which starts with Animalia or biota and all the way down), I would still have to expand everything to see which labels are muted and which are not. Am I understanding this correctly?

**Comment by mzur** [comment]
This is correct. So you would prefer a flat list of labels instead, hiding all (parent) labels that have no annotations?

**Comment by agnanas** [comment]
Ideally, I'd like to keep the label trees themselves hierarchical and have the tree visible for any relabeling if needed. But just to visualise all the boxes containing a label, a tab similar to what you have in the video or image mode for annotations would be great in Largo as well. Something like shown below, that way, I know the labels for my current volume and I can go labelwise to see all annotations and can decide or deletions or relabeling. 
![image](https://github.com/biigle/largo/assets/108525376/9eda8ed7-e2d2-494e-9921-d0f2386a03f3)

**Comment by mzur** [comment]
That's it! So to sum it up:

- Add a new sidebar tab with a "list" icon.
- When the tab is opened, a database query is executed to fetch the list of labels with annotations and the annotation count. The query may take a few seconds.
- The sidebar tab then shows the list of labels as in the screenshot above (without the filter component). A click on a label selects the label.

This solves multiple issues at the same time:
- The tab is closed by default so the expensive database query is only executed when needed.
- The user can choose between the usual hierarchical label trees and the flat list of filtered labels.
- We can show the annotation count for each label (resolves https://github.com/biigle/largo/issues/98).

Any objections?

**Comment by agnanas** [comment]
Spot on! No objections what to ever
